### ISSUE TITLE
Step by step guide for absolute beginners to contribute

### ISSUE THREAD

**Issue by ankit-saraf** [anitab-org/documentation#22]
## Step by step guide for absolute beginners to contribute.
As a beginner,
I need a complete beginner's guide on how to proceed on their first issue,
so that I can break the hesitation that the beginners have which makes them reluctant to contribute.

## Required items:
1. A BEGINNER_GUIDE.md file with step by step guide.
2. Step by step git commands for setting repository locally.
3. Creating a pull request.
4. Relevant links to documentation.
5. What to do if get stuck.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
12 hours

**Comment by ankit-saraf** [comment]
I would love to work on it

**Comment by ankit-saraf** [comment]
Please assign it to me

**Comment by ankit-saraf** [comment]
Pls add the following labels :
Category: Documentation
Open Source Hack

**Comment by vj-codes** [comment]
This repo is not for OSH so can't add that label

**Comment by ankit-saraf** [comment]
@vj-codes can we have it in VMS?

**Comment by ankitapuri** [comment]
Can I work on this if its available

**Comment by ankitapuri** [comment]
@vj-codes  please assign this to me .

**Comment by madhhuurrii** [comment]
@vj-codes Can u please assign me ?

**Comment by Code414** [comment]
Please assign this issue to me. I want to write the documentation for it. Thanks :)

**Comment by ayushi1109** [comment]
could you please assign this task to me ??

**Comment by nandini45** [comment]
@isabelcosta do you approve of  this issue??

**Comment by isabelcosta** [comment]
@nandini45 you are correct this is not approved yet, I asked for help [here](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentation/topic/anitab-org.2Fdocumentation/near/212465514) from ambassadors.

**Comment by unnati914** [comment]
can i work on this issue

**Comment by nandini45** [comment]
@unnati914  this issue is not available for a new member in the community. it is basically for a person who has been contributing to the community for some while. the aim of this issue is to increase more community engagements.
 you can look more issues with the available tag.

**Comment by unnati914** [comment]
ok thanks!
